<?xml version="1.0"?>
<launch>
	<param name="use_sim" value="false" />

	<env name="ROSCONSOLE_CONFIG_FILE" value="$(find limb_manipulation)/config/custom_rosconsole.conf"/>
	
	<include file="$(find fetch_moveit_config)/launch/move_group.launch" />

	<arg name="use_rviz" default="true" />
	<node pkg="rviz" type="rviz" name="rviz"
    	args="-d $(find limb_manipulation)/config/limb_manipulation_ar_config.rviz"
    	if="$(arg use_rviz)" />

	<include file="$(find limb_manipulation)/launch/ar.launch">
    	<arg name="cam_image_topic" default="/head_camera/depth_registered/points" />
	</include>
</launch>
